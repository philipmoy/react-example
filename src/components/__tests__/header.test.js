import React from 'react';
import { shallow } from 'enzyme';

import AppHeader from '../header';

describe('AppHeader', () => {
  it('renders correctly', () => {
    const wrapper = shallow(
      <AppHeader />,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
