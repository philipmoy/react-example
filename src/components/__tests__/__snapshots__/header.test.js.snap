// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppHeader renders correctly 1`] = `
<header
  className="header header--app"
>
  <img
    alt="logo"
    className="header--app__logo"
    src="/images/logo.svg"
  />
</header>
`;
